msg.c

```diff
@@ -26,136 +26,130 @@
 
 static u32 msg_csum(msg_t *msg)
 {
-    u32 k;
-    u16 csum;
-
-    csum = 0;
-    for (k = 0; k < MSG_HDRSZ + msg_getlen(msg); k++) {
-        if (k >= sizeof(msg->csum))
-            csum += *((u8 *)msg + k);
-    }
-    csum += msg_getseq(msg);
-    return csum;
+	u32 k;
+	u16 csum;
+
+	csum = 0;
+	for (k = 0; k < MSG_HDRSZ + msg_getlen(msg); k++) {
+		if (k >= sizeof(msg->csum))
+			csum += *((u8 *)msg + k);
+	}
```

Comment:
```suggestion
	for (k = sizeof(msg->csum); k < MSG_HDRSZ + msg_getlen(msg); k++) {
		csum += *((u8 *)msg + k);
	}
```
I was going to create PR, but if you are changing this file then by the way this change. 
It will be faster.

Reply:
Thanks for your note, it is obviously correct. 
However, this PR is related to add USB driver and modules associated with it and its size has grown significantly due to many bindings.
I have added your suggestion but if changes are not strictly related to current pull request, I would like to ask you to create a new PR next time.